modules/lightning_telemetry/lightning.services.yml

```diff
@@ -0,0 +1,4 @@
+services:
```

Comment:
This file should not exist. Since it's in the lightning_telemetry module, it will never be read. The services file should be lightning_telemetry.services.yml.

Reply:
I got a CI error when I left this file out.